docs/data/charts/hooks/use-series.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Charts - useSeries
+productId: x-charts
+---
+
+# useSeries
+
+<p class="description">Access raw series data for all chart types.</p>
+
+The `useXxxSeries` hook provides access to specific series data for a particular chart type.
+
+## Usage
+
+It is recommended to use the specific series hooks (e.g., `useBarSeries`, `useLineSeries`) when working with a specific chart type, as their API is easier to use.
+
+```js
+import { useBarSeries, useLineSeries } from '@mui/x-charts/hooks';
+
+function CustomComponent() {
+  const barSeries = useBarSeries();
+  const lineSeries = useLineSeries();
+  // barSeries: Object containing bar chart series data
+  // lineSeries: Object containing line chart series data
+}
+```
+
```

Comment:
````suggestion
```

You can also pick specific series by either providing the series id as a parameter, or an array of series ids to get.

```js
const barSeries = useBarSeries('id1'); // 

const barSeries = useBarSeries(['id1', 'id2']);
```

````